### DISCUSSION TITLE
Registering Parents accounts for Families.

### DISCUSSION THREAD

**Discussion by NaveedAhmadHematmal** [VDTS/CodeX.VaxineSolution#41] [Ideas]
1: Mobilizer requests admin or 2: Mobilizer requests Supervisor or 3: supervisor requests admin to register a parent-role account and add his family to it.
Which way is more secure, logical and applicable?

**Comment by SaeedaRasuly** [comment]
I think the mobilizer should request supervisor for registering parent role and adding his family.

**Comment by SaeedaRasuly** [comment]
Cause supervisor is the responsible person for such tasks

**Comment by NaveedAhmadHematmal** [comment]
I think so, lets ask @YasinZahin what he thinks.

**Comment by YasinZahin** [comment]
I think mobilizer doesn't need to request someone else for adding parents. This should be the authority of mobilizer.

**Reply by NaveedAhmadHematmal** [reply]
As @SaeedaRasuly mentioned, "Cause supervisor is the responsible person for such tasks".
Mobilizers jobs are limited to vaccine periods, supervisors jobs are not. All such decisions done by supervisor because they are more involve with people than mobilizers. And there is privacy issue, that should be control by supervisors or admin.

Either Mobilizer should request supervisors or supervisors should register parents without involving mobilizers in the process.

**Reply by SaeedaRasuly** [reply]
Mobilizers are only responsible for conducting field visits and registering data, The supervisor is the person who have the authorities of adding deleting and customizing

**Reply by SaeedaRasuly** [reply]
As with the organization I used to work in previously, I was conducting field visits and filling the forms, and registering data; my supervisor was the person who had more authority on customization and I was only doing the tasks that were assigned for me by my direct supervisor.

**Reply by NaveedAhmadHematmal** [reply]
@SaeedaRasuly Got it, we will implement adding parents' accounts in the supervisor app.
